EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001425	This pocket was buried in the spot where the smell was coming from?{nl}Let's see...
+QUEST_LV_0300_20160310_001426	The number of monsters increased a lot so the number of wounded people is increasing.{nl}We don't have enough medicine...
+QUEST_LV_0300_20160310_001427	If it weren't you...
+QUEST_LV_0300_20160310_001428	I knew it was strange.{nl}It was a smell we couldn't understand.
+QUEST_LV_0300_20160310_001429	It is difficult.{nl}The deadline is coming, but we can't grab the corals...
 QUEST_LV_0300_20160310_001430	Putting all of us in danger for such petty motives, truly there can be no eviler act than that.{nl}Thank heavens, you were here to save the town.
-QUEST_LV_0300_20160310_001431	I am worried since the number of monsters increased a lot recently.{nl}They never came down this far..
-QUEST_LV_0300_20160310_001432	He didn't even bat an eye when they are injured.{nl}I hope the next lieutenant would be nicer.
+QUEST_LV_0300_20160310_001431	I am worried since the number of monsters increased a lot recently.{nl}They never came down this far...
+QUEST_LV_0300_20160310_001432	He didn't even bat an eye when they are injured.{nl}I hope the next lieutenant is nicer.
 QUEST_LV_0300_20160310_001433	The villagers are giving up on their work because the number of monsters increased a lot.{nl}I know it's an order, but it is so unfortunate.
-QUEST_LV_0300_20160310_001434	No matter how hard we have tried.{nl}I cannot accept that we have to leave this place..
-QUEST_LV_0300_20160310_001435	Thank  you!{nl}I can work again because of you!
-QUEST_LV_0300_20160310_001436	It was strange.{nl}But, we waited because we thought there must be some reason��{nl}
-QUEST_LV_0300_20160310_001437	We were being used, but we should regret the fact that we just waited for the order.{nl}I want to apologize to the villagers if I have opportunities.
-QUEST_LV_0300_20160310_001438	I can't do this anymore.{nl}I tried to start again here with everything I've got.. But, I don't know what happened.
+QUEST_LV_0300_20160310_001434	No matter how hard we tried.{nl}I cannot accept that we have to leave this place...
+QUEST_LV_0300_20160310_001435	Thank you!{nl}I can work again because of you!
+QUEST_LV_0300_20160310_001436	It was strange.{nl}We waited because we thought there must be some reason...{nl}
+QUEST_LV_0300_20160310_001437	We were being used, but we should regret the fact that we just waited for the order.{nl}I want to apologize to the villagers if I have the opportunity.
+QUEST_LV_0300_20160310_001438	I can't do this anymore.{nl}I tried to start again here with everything I've got... But, I don't know what happened.
 QUEST_LV_0300_20160310_001439	Thank you for helping me go back to the workplace.{nl}The humans are more scary than the monsters sometimes.
```

Comment:
"Thank you for helping me get back to work.{nl}Humans are more scary than monsters, sometimes."

Assuming a human is saying these things ^